platform-configs/protectli-vp4670.robot

```diff
@@ -66,15 +66,15 @@ ${DEVICE_USB_ROOT_PROMPT}=                          root@user-VP4670:/home/user#
 
 ${DEVICE_WINDOWS_USERNAME}=                         user
 ${DEVICE_WINDOWS_PASSWORD}=                         windows
-${DEVICE_UBUNTU_USERNAME}=                          user
-${DEVICE_UBUNTU_PASSWORD}=                          ubuntu
-${DEVICE_UBUNTU_USER_PROMPT}=                       user@user-VP4670:~$
-${DEVICE_UBUNTU_ROOT_PROMPT}=                       root@user-VP4670:/home/user#
+${UBUNTU_USERNAME}=                                 ubuntu
+${UBUNTU_PASSWORD}=                                 ubuntu
+${UBUNTU_USER_PROMPT}=                              ubuntu@3mdeb:~$
+${UBUNTU_ROOT_PROMPT}=                              root@3mdeb:/home/ubuntu#
 ${3_MDEB_WIFI_NETWORK}=                             3mdeb_abr
 
 ${DMIDECODE_SERIAL_NUMBER}=                         N/A
 ${DMIDECODE_FIRMWARE_VERSION}=                      Dasharo (coreboot+UEFI) v1.0.19
-${DMIDECODE_PRODUCT_NAME}=                          VP4670
+${DMIDECODE_PRODUCT_NAME}=                          VP4650
```

Comment:
@johanes2115 We have 4650 name in VP4670 config?

Reply:
fixed in ea5eef844a1d6abd720f7551ec7dfcdd66fa605f